spec/io/sql_data_source_spec.rb

```diff
@@ -62,5 +62,22 @@
         }.to raise_error(ArgumentError)
       end
     end
+
+    context 'with path to sqlite3 file' do
+      subject(:df) { Daru::IO::SqlDataSource.make_dataframe(db_name, query) }
+      it { is_expected.to be_a(Daru::DataFrame) }
+      it { expect(df.nrows).to eq 2 }
+      it { expect(df.row[0][:id]).to eq 1 }
+      it { expect(df.row[0][:name]).to eq 'Homer' }
```

Comment:
@zverok I did not know that it could be written like that. Is it reasonable to open an issue to begin updating the tests throughout the library. I would be up to doing that next since I need to let the new syntax sink in.